### PR TITLE
Feature/styling components

### PR DESCRIPTION
**Pull request by angarita-dev**
# Hi :rocket: 

On this feature I introduced styling for the calculator.

### Introduced changes:
* Added `node-sass` package.
* Added a `color` prop to the Button component.
* Added a `wide` prop to the Button component.
* Added variables file with the colors used.
* Added styling for every component.

@codingAngarita :wave:

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <Button buttonName="-" backgroundColor="#ff9d14" />
+      </div>
+      <div id="g4-buttons" className="button-row">
+        <Button buttonName="1" />
+        <Button buttonName="2" />
+        <Button buttonName="3" />
+        <Button buttonName="+" backgroundColor="#ff9d14" />
+      </div>
+      <div id="g5-buttons" className="button-row">
+        <Button buttonName="0" />
```
Comment (shshamim63) [inline src/components/ButtonPanel.js]: You need the pass the prop wide with Button '0' component.

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## :recycle: Changes Required :recycle: 
___
Hi @codingAngarita :wave: 
Awesome job, the design actually looks way closer to the given template. Just a simple issue blocked me from approving this milestone. Please resolve it and send another review request.
___
### :spades: Mandatory Changes :spades: 
___
- You need to pass the wide prop in the Button '0' component. Cause without that it is not going to apply the width that it should get. Currently, it looks like this.
image
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Comment by angarita-dev** [comment]
# Hi! :robot: 

### New changes:
* Added the wide prop for button `0` **_oops!_**

@codingAngarita :wave:

**Review by suretrust — APPROVED** [review]
Hi @codingAngarita, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future milestones and happy coding!_ 🎉

**Pull request merged**